Plk.Blazor.DragDrop/Dropzone.razor

```diff
@@ -236,6 +253,12 @@
     /// </summary>
     [Parameter] public Func<TItem, bool> Accepts { get; set; }
 
+
+    /// <summary>
+    /// Allows to pass a delegate which executes if something is dropped and decides if the item is accepted
```

Comment:
comment is wrong , I assume it should be something like ... which excecutes before something is dragged and decides if dragging is allowed

Reply:
It actually makes the determination at render rather than trying to wind in with the drag event.